src/CMakeLists.txt

```diff
@@ -1,101 +1,47 @@
 # TODO: Break this cmake(app_base target) into smaller libraries with their own cmakes, e.g. network, chain, dag, etc... could be separate libraries
 
 set(SOURCES
-        # ---- public headers -----
-        util/util.hpp
-        node/full_node.hpp
-        transaction_manager/transaction_manager.hpp
-        storage/db_storage.hpp
-        dag/dag.hpp
-        consensus/pbft_chain.hpp
-        aleth/node_api.hpp
-        aleth/database.hpp
-        aleth/state_api.hpp
-        aleth/dummy_eth_apis.hpp
-        consensus/block_proposer.hpp
-        transaction_manager/transaction_status.hpp
-        chain/chain_config.hpp
-        consensus/vrf_wrapper.hpp
-        node/executor.hpp
-        network/network.hpp
-        common/static_init.hpp
-        dag/vdf_sortition.hpp
-        chain/final_chain.hpp
-        consensus/pbft_config.hpp
-        network/taraxa_capability.hpp
-        util/exit_stack.hpp
-        util/encoding_rlp.hpp
-        util/range_view.hpp
-        util/lazy.hpp
-        config/config.hpp
-        node/replay_protection_service.hpp
-        common/types.hpp
-        dag/dag_block.hpp
-        network/packets_stats.hpp
-        network/rpc/Taraxa.h
-        network/rpc/Test.h
-        network/rpc/TaraxaClient.h
-        network/rpc/NetClient.h
-        network/rpc/Net.h
-        network/rpc/NetFace.h
-        network/rpc/TaraxaFace.h
-        network/rpc/TestFace.h
-        network/rpc/RpcServer.h
-        network/rpc/WSServer.h
-        network/rpc/TestClient.h
-        transaction_manager/transaction_order_manager.hpp
-        consensus/vote.hpp
-        transaction_manager/transaction.hpp
-        transaction_manager/transaction_queue.hpp
-        logger/logger_config.hpp
-        logger/log.hpp
-        chain/state_api.hpp
-        dag/dag_block_manager.hpp
-        dag/proposal_period_levels_map.hpp
-        consensus/pbft_manager.hpp
```

Comment:
put list of headers back

Reply:
Ok, but is there any particular reason for this? As far as I know there's no effect from listing headers as compilation units

Reply:
yes there is. To see headers for IDE-generated projects you have to provide list of headers. Just put it back
https://stackoverflow.com/questions/36174499/why-add-header-files-into-add-library-add-executable-command-in-cmake

Reply:
done